tools/healthcheck/healthcheck.py

```diff
@@ -0,0 +1,32 @@
+from requests_hawk import HawkAuth
+import requests
+import os
+
+
+def healthcheck(event, context):
+    hawk_auth = HawkAuth(
+        id=os.environ['HAWK_ID'],
+        key=os.environ['HAWK_KEY'])
+
+    ip = u"127.0.0.1"
```

Comment:
Like I said before: use a class E IP that's specifically meant for research and is non routable. I don't want to mess with localhost in the event it doesn't get cleaned up from a DB.